templates/accordion.hbs

```diff
@@ -13,10 +13,12 @@
                 <div class="accordion-item-body-inner">
                     {{{a11y_text body}}}
                     {{#if _graphic}}
+                    {{#if _graphic.src}}
```

Comment:
any risk of an error if _graphic is also null? should it be `{{#if (_graphic && graphic.src) }}`? (dunno if that's possible in handlebars)